doc/example/model_selection.ipynb

```diff
@@ -0,0 +1,828 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Model Selection"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example Model\n",
+    "In this example, measurement data are simulation results with the \"true\" model. The comprehensive model includes additional behavior involving a third parameter. Hence, during model selection, models with $\\theta_3=0$ should be preferred.\n",
+    "\n",
```

Comment:
if you wanna be fancy, you can add an RST cell with links to the main classes, such as `'Problem <pypesto.select.Problem>'`, see e.g. https://pyabc.readthedocs.io/en/develop/examples/optimal_threshold.html.